README.md

```diff
@@ -0,0 +1,27 @@
+# OnlyofficeLoggerHelper
+
+Helper gem for logging info messages in Ruby
```

Comment:
You can take description here
https://github.com/onlyoffice-testing-robot/onlyoffice_webdriver_wrapper/blob/master/onlyoffice_webdriver_wrapper.gemspec#L13